tests/test_template_init.py

```diff
@@ -144,29 +148,29 @@ def test_template_suite(
     # Run the local test suite.
     try:
         subprocess.run(
-            "hatch run install:check",
-            cwd=project_dir,
-            check=True,
-            shell=True,
-        )
-        subprocess.run(
-            f"hatch run +py={sys.version_info.major}.{sys.version_info.minor} test:run",
-            cwd=project_dir,
-            check=True,
-            shell=True,
-        )
-        subprocess.run(
-            "hatch run style:check",
-            cwd=project_dir,
-            check=True,
-            shell=True,
-        )
-        subprocess.run(
-            "hatch run audit:check",
+            "hatch build --clean",
             cwd=project_dir,
             check=True,
             shell=True,
         )
+        # subprocess.run(
```

Comment:
Commented out because we have to rethink the flow of tests now if we don't assume the user is using hatch environments.